ci/container-images-addl-tags.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# This script produces output in the form of
+#
+#     $ REMOTE=awelzel ./ci/container-images-addl-tags.sh v7.0.5
+#     ADDITIONAL_MANIFEST_TAGS= lts 7.0 latest
```

Comment:
Puh, you're right. I guess the `$CIRRUS_ENV` file works in a way that quotes aren't needed, otherwise would've failed previously. But yes, one can't just export the output of the script.